src/lib/code-sample-editor.ts

```diff
@@ -83,8 +84,26 @@ export class CodeSampleEditor extends LitElement {
   @property({attribute: false})
   files?: SampleFile[];
 
-  // TODO: make a public property/method to select a file
-  @property({attribute: false})
+  /**
+   * The name of the project file that is currently being displayed.
+   */
+  @property()
+  filename?: string;
+
+  /**
+   * If true, don't display the top file-picker. Default: false (visible).
+   */
+  @property({type: Boolean, attribute: 'no-file-picker'})
+  noFilePicker = false;
+
+  /**
+   * If true, don't display the left-hand-side gutter with line numbers. Default
+   * false (visible).
+   */
+  @property({type: Boolean, attribute: 'no-line-numbers'})
+  noLineNumbers = false;
```

Comment:
I think you could make this `lineNumbers` and default to `false`.

Reply:
Done, but made it on in `<code-sample>`.